### PR TITLE
Added limitation for inputs for a workflow_dispatch event

### LINKED ISSUE #15710: Add notice about limits for inputs` for a `workflow_dispatch` event
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

I am trying to configure Github Action with more than 10 `inputs` for a `workflow_dispatch` event and I am getting the  error:

**[Invalid workflow file: .github/workflows/create_feature_branch.yml#L1](https://github.com/Lolo-Health/lolo-infrastructure/actions/runs/1878343346/workflow)
you may only define up to 10 `inputs` for a `workflow_dispatch` event**

### What changes are you suggesting?

We definitely need to add notice about such limit and increase it

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by bandantonio**
### Why:

Closes #15710 

### What's being changed: documentation

workflow_dispatch

### Check off the following:

- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@bandantonio 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by ethomson** [comment]
Looks good to me, thanks for the fix @bandantonio

File: content/actions/managing-workflow-runs/manually-running-a-workflow.md [hunk 1]
```diff
@@ -73,4 +73,10 @@ gh run watch
 
 When using the REST API, you configure the `inputs` and `ref` as request body parameters. If the inputs are omitted, the default values defined in the workflow file are used.
 
+{% note %}
+
+**Note:** You can define up to 10 `inputs` for a `workflow_dispatch` event
```
Comment (myarb) [inline content/actions/managing-workflow-runs/manually-running-a-workflow.md:78]:
```suggestion
**Note:** You can define up to 10 `inputs` for a `workflow_dispatch` event.
```

**Review by myarb — DISMISSED** [review]
This has been approved by an SME, so I'm making a tweak and will merge soon 👍🏼

**Review by myarb — APPROVED** [review]
Approving and merging shortly 🎉

**Pull request merged**